RadixWallet/RadixConnect/RadixConnectModels/Types.swift

```diff
@@ -13,8 +13,76 @@ extension PeerConnectionID: Sendable {}
 public enum ConnectionPasswordTag {}
 public typealias ConnectionPassword = Tagged<ConnectionPasswordTag, HexCodable32Bytes>
 
+extension ConnectionPassword {
+	var messageToHash: Data {
+		let prefix = Data("L".utf8)
+		let passwordData = self.data.data
+		var messageData = Data()
+		messageData.append(prefix)
+		messageData.append(passwordData)
+
+		return Data(messageData)
+	}
+}
+
 #if DEBUG
 extension ConnectionPassword {
 	public static let placeholder = try! Self(.init(.deadbeef32Bytes))
 }
 #endif // DEBUG
+
+// MARK: - CEPublicKeyTag
+public enum CEPublicKeyTag {}
+public typealias CEPublicKey = Tagged<CEPublicKeyTag, HexCodable32Bytes>
+
+#if DEBUG
+extension CEPublicKey {
+	public static let placeholder = try! Self(.init(.deadbeef32Bytes))
+}
+#endif // DEBUG
+
+// MARK: - CESignatureTag
+public enum CESignatureTag {}
+public typealias CESignature = Tagged<CESignatureTag, HexCodable>
+
+#if DEBUG
+extension CESignature {
+	public static let placeholder = try! Self(.deadbeef32Bytes)
+}
+#endif // DEBUG
+
+// MARK: - LinkConnectionQRData
+public struct LinkConnectionQRData: Sendable, Hashable, Decodable {
+	public let purpose: ConnectionPurpose
+	public let password: ConnectionPassword
+	public let publicKey: CEPublicKey
+	public let signature: CESignature
```

Comment:
Please document that the signature was produced by CE having signed the hash of the `password` with the privatekey of the `publicKey` :)

Reply:
You can also point to `ConnectionPassword.messageToHash`, that CE does exactly the sam thing on its end.